isdown/manifest.json

```
@@ -0,0 +1,57 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "98d8eda3-7bc5-4067-8ff6-63d11d0132e2",
```

```suggestion
  "app_uuid": "22560cfe-27cc-492f-a978-64dfcdc3b3c0",
```
Please update this field to the new app_uuid.